### PR TITLE
*: Clear the name of some field  in explain result.

### PR DESCRIPTION
**Pull request by LittleFall**
### What problem does this PR solve? 

Fixed the third problem int [issue/14603](https://github.com/pingcap/tidb/issues/14603):
- For `EXPLAIN` and `EXPLAIN ANALYZE` result, change column name `count` to `estRows`(estimated rows).
- For `EXPLAIN ANALYZE` result, add a new column `actRows`(actual rows), and put it near by `estRows`.

For example, execute query
```sql
DROP TABLE IF EXISTS t1;
CREATE TABLE t1 (
	id VARCHAR(31),
	name VARCHAR(50),
	age INT,
	KEY idx(id)
);
INSERT INTO t1 VALUES ("001", "yuqiu", 18);
EXPLAIN ANALYZE SELECT * FROM t1 USE INDEX(idx);
```

The result used to be:
```
+-------------------------------+-------+-----------+----------------------------------------------------+-------------------------------------------------------------------------------+---------------+------+
| id                            | count | task      | operator info                                      | execution info                                                                | memory        | disk |
+-------------------------------+-------+-----------+----------------------------------------------------+-------------------------------------------------------------------------------+---------------+------+
| IndexLookUp_6                 | 1.00  | root      |                                                    | time:1.238223ms, loops:2, rows:1, rpc num: 1, rpc time:322.52µs, proc keys:0  | 9.34765625 KB | N/A  |
| ├─IndexFullScan_4(Build)      | 1.00  | cop[tikv] | table:t1, index:id, keep order:false, stats:pseudo | time:153.145µs, loops:2, rows:1                                               | N/A           | N/A  |
| └─TableRowIDScan_5(Probe)     | 1.00  | cop[tikv] | table:t1, keep order:false, stats:pseudo           | time:89.884µs, loops:2, rows:1                                                | N/A           | N/A  |
+-------------------------------+-------+-----------+----------------------------------------------------+-------------------------------------------------------------------------------+---------------+------+
```

And now:
```
+-------------------------------+---------+---------+-----------+----------------------------------------------------+--------------------------------------------------------------------------------+---------------+------+
| id                            | estRows | actRows | task      | operator info                                      | execution info                                                                 | memory        | disk |
+-------------------------------+---------+---------+-----------+----------------------------------------------------+--------------------------------------------------------------------------------+---------------+------+
| IndexLookUp_6                 | 1.00    | 1       | root      |                                                    | time:896.517µs, loops:2, rows:1, rpc num: 1, rpc time:216.798µs, proc keys:0   | 9.34765625 KB | N/A  |
| ├─IndexFullScan_4(Build)      | 1.00    | 1       | cop[tikv] | table:t1, index:id, keep order:false, stats:pseudo | time:79.388µs, loops:2, rows:1                                                 | N/A           | N/A  |
| └─TableRowIDScan_5(Probe)     | 1.00    | 1       | cop[tikv] | table:t1, keep order:false, stats:pseudo           | time:51.716µs, loops:2, rows:1                                                 | N/A           | N/A  |
+-------------------------------+---------+---------+-----------+----------------------------------------------------+--------------------------------------------------------------------------------+---------------+------+
```

### What is changed and how it works?

The value of `actRows` equals to `rows` in `execute info`.
Referred to [Understand the Query Execution Plan](https://pingcap.com/docs/stable/reference/performance/understanding-the-query-execution-plan/#explain-analyze-output-format)

### Check List 

Tests 

 - Unit test
 - Integration test
 - No code

Code changes

 - Has exported function/method change
 - Has persistent data change

Related changes

 - Need to update the documentation
[Understand the Query Execution Plan](https://pingcap.com/docs/stable/reference/performance/understanding-the-query-execution-plan/#explain-analyze-output-format)
 - Update tidb-test with [pr/997](https://github.com/pingcap/tidb-test/pull/997)

Release note

- For `EXPLAIN` and `EXPLAIN ANALYZE` result, change column name `count` to `estRows`(estimated rows).
- For `EXPLAIN ANALYZE` result, add a new column `actRows`(actual rows), and put it near by `estRows`.

### CODE REVIEW

File: planner/core/explain.go [hunk 1]
```diff
@@ -245,7 +245,7 @@ func (p *PhysicalIndexReader) ExplainNormalizedInfo() string {
 func (p *PhysicalIndexLookUpReader) ExplainInfo() string {
 	// The children can be inferred by the relation symbol.
 	if p.PushedLimit != nil {
-		return fmt.Sprintf("limit embedded(offset:%v, count:%v)", p.PushedLimit.Offset, p.PushedLimit.Count)
+		return fmt.Sprintf("limit embedded(offset:%v, estRows:%v)", p.PushedLimit.Offset, p.PushedLimit.Count)
```
Comment (zz-jason) [inline planner/core/explain.go]: no, this should be `count`

File: planner/core/explain.go [hunk 2]
```diff
@@ -293,7 +293,7 @@ func (p *PhysicalSort) ExplainInfo() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalLimit) ExplainInfo() string {
-	return fmt.Sprintf("offset:%v, count:%v", p.Offset, p.Count)
+	return fmt.Sprintf("offset:%v, estRows:%v", p.Offset, p.Count)
```
Comment (zz-jason) [inline planner/core/explain.go]: ditto

File: planner/core/explain.go [hunk 3]
```diff
@@ -472,7 +472,7 @@ func (p *PhysicalMergeJoin) ExplainNormalizedInfo() string {
 func (p *PhysicalTopN) ExplainInfo() string {
 	buffer := bytes.NewBufferString("")
 	buffer = explainByItems(buffer, p.ByItems)
-	fmt.Fprintf(buffer, ", offset:%v, count:%v", p.Offset, p.Count)
+	fmt.Fprintf(buffer, ", offset:%v, estRows:%v", p.Offset, p.Count)
```
Comment (zz-jason) [inline planner/core/explain.go]: ditto

Comment (LittleFall) [reply planner/core/explain.go]: So the strings of `ExplainInfo()`:
- In `PhysicalIndexLookUpReader`, `PhysicalLimit` or `PhysicalTopN`, it should be `count`.
- In `LogicalTopN`, `LogicalLimit`, it should be `estRows`.

Is this right?

File: planner/core/explain.go [hunk 4]
```diff
@@ -703,13 +703,13 @@ func (p *LogicalSort) ExplainInfo() string {
 func (p *LogicalTopN) ExplainInfo() string {
 	buffer := bytes.NewBufferString("")
 	buffer = explainByItems(buffer, p.ByItems)
-	fmt.Fprintf(buffer, ", offset:%v, count:%v", p.Offset, p.Count)
+	fmt.Fprintf(buffer, ", offset:%v, estRows:%v", p.Offset, p.Count)
```
Comment (zz-jason) [inline planner/core/explain.go]: it should be `count`

File: planner/core/explain.go [hunk 5]
```diff
@@ -703,13 +703,13 @@ func (p *LogicalSort) ExplainInfo() string {
...
 	return buffer.String()
 }
 
 // ExplainInfo implements Plan interface.
 func (p *LogicalLimit) ExplainInfo() string {
-	return fmt.Sprintf("offset:%v, count:%v", p.Offset, p.Count)
+	return fmt.Sprintf("offset:%v, estRows:%v", p.Offset, p.Count)
```
Comment (zz-jason) [inline planner/core/explain.go]: ditto

File: planner/core/common_plans.go [hunk 6]
```diff
@@ -884,6 +892,18 @@ func (e *Explain) prepareOperatorInfo(p Plan, taskType, driverSide, indent strin
 		} else {
 			row = append(row, "N/A")
 		}
+
+		// Calculate 'actRows'(actual rows) and put it next by 'estRows'(estimate rows).
+		var actRows string
+		if totalLoops == 0 {
+			actRows = "N/A"
+		} else {
+			actRows = fmt.Sprint(float64(totalRows) / float64(totalLoops))
```
Comment (zz-jason) [inline planner/core/common_plans.go]: it should be `totalRows`, why dividing it with `totalLoops`?

Comment (zz-jason) [inline planner/core/common_plans.go]: it should be `totalRows`, why dividing it with `totalLoops`?

**Pull request merged**